### PR TITLE
ghost text improvements

### PR DESCRIPTION
**Pull request by hediet**

### CODE REVIEW

File: src/vs/editor/contrib/inlineCompletions/ghostTextController.ts [hunk 1]
```diff
@@ -173,6 +187,50 @@ registerEditorCommand(new GhostTextCommand({
...
+				weight: 100,
+				primary: KeyMod.CtrlCmd | KeyMod.Alt | KeyMod.Shift | KeyCode.LeftArrow,
+			},
+		});
+	}
+
+	public async run(accessor: ServicesAccessor | undefined, editor: ICodeEditor): Promise<void> {
+		const controller = GhostTextController.get(editor);
+		if (controller) {
+			controller.showNextInlineCompletion();
```
Comment (Krzysztof-Cieslak) [inline src/vs/editor/contrib/inlineCompletions/ghostTextController.ts:229]: @hediet, this should be `controller. showPreviousInlineCompletion();`, shouldn't it?

**Comment by Krzysztof-Cieslak** [comment]
@hediet, just for confirmation - as far as I understand, this PR has added cycling when all the results are returned from the initial API call, not the partial results flow we've discussed yesterday?

File: src/vs/editor/contrib/inlineCompletions/ghostTextController.ts (hunk 1 above)
Comment (hediet) [reply src/vs/editor/contrib/inlineCompletions/ghostTextController.ts:229]: Already fixed (will be available tomorrow). In my test I only had two suggestions, so previous and next did the same.

**Comment by hediet** [comment]
> @hediet, just for confirmation - as far as I understand, this PR has added cycling when all the results are returned from the initial API call, not the partial results flow we've discussed yesterday?

I didn't manage to implement that logic yesterday. It will be available in tomorrows insider build.

**Comment by Krzysztof-Cieslak** [comment]
Thanks a lot ❤️

**Pull request merged**